definitions/variable/afolu/food_system.yaml

```diff
@@ -0,0 +1,92 @@
+- Food Availability [per capita]:
```

Comment:
Please name this file as `food-system.yaml` for consistency. And if you think it's necessary to move delete the file `food.yaml` and move "Food Demand" somewhere else, why not here?

Reply:
Done.